src/python/examples/memory_growth_test.py

```diff
@@ -64,9 +61,11 @@
 
     for i in range(FLAGS.repetitions):
         if FLAGS.protocol.lower() != "grpc":
+            import tritonclient.http as httpclient
```

Comment:
move the import and conditional to right before the loop.

I think you can even take a step further and use partial function / lambda to unify the HTTP/GRPC handling, to avoid protocol check in multiple places, something like:
```
if grpc:
    create_client = partial(grpcclient.InferenceServerClient, url="localhost:8001", verbose=FLAGS.verbose)
    ... # I/O unification
else:
    create_client = partial(httpclient.InferenceServerClient, url="localhost:8000", verbose=FLAGS.verbose)
    ... # I/O unification
```
And later just
```
for repetitions:
    triton_client = create_client()
    ... # I/O
```